### DISCUSSION TITLE
Default resolution for Privacy Mode 2

### DISCUSSION THREAD

**Discussion by SleepyPrince** [rustdesk/rustdesk#8946] [Feature Request] [answered]
For Privacy Mode 2 on Windows host and Windows client, the resolution always reset to 1024*768 on each activation. Is it possible to add an option to set the default resolution (select from list or match client resolution)?

**Comment by rustdesk** [comment]
@fufesou

**Reply by rustdesk** [reply]
The default resolution not match client resolution?

**Reply by fufesou** [reply]
Yes. It is just plugged in. No further operations.

**Comment by fufesou** [comment]
I'll fix it.

**Comment by fufesou** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/pull/8992

**Reply by fufesou** [reply]
> add an option to set the default resolution (select from list or match client resolution)

I do this now for simplicity.

Adding custom resolutions is also unstable.
There are not many resolutions supported by the control side. eg. The idd says it supports `2560x1440`, but I got `DISP_CHANGE_BADMODE` with that mode.

Maybe I'll set the `DPI` and client resolution in the future.

**Reply by ZhaoZelin2000** [reply]
> > 添加设置默认分辨率的选项（从列表中选择或匹配客户端分辨率）
> 
> 为了简单起见，我现在这样做。
> 
> 添加自定义分辨率也不稳定。控制端支持的分辨率不多。例如。idd 说它支持 ，但我选择了这种模式。`2560x1440``DISP_CHANGE_BADMODE`
> 
> 也许我将来会设置和客户端解决方案。`DPI`

I am looking forward to automatic resolution matching. I am currently using Android 1.4.3 to remotely connect to Windows 11 Rustdesk 1.4.3, and every time I turn on Privacy Mode 2, I need to manually adjust the resolution. If the resolution cannot be automatically matched at present, I hope that the device inserted into the virtual display can have different IDs according to different control terminals, because Windows will remember the resolution and dpi manually set last time.
